apps/ideaspace/src/components/common/CommonStyles.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+export const HeadWapper = styled.div`
+  background-color: #FFFFFF;
+  padding: 6rem 1rem 4rem 1rem;
+  text-align: center;
+
+  @media (max-width: 1712px) {
+    padding: 3rem 1rem 2rem 1rem;
+  }
+
+  @media (max-width: 529px) {
+    padding: 5rem 1rem 1rem 1rem;
+  }
+`;
+
+export const Headline = styled.div`
+  font-family: 'Abel';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 4rem;
+  line-height: 68px;
+  text-align: center;
+  letter-spacing: -0.02em;
+  color: #1C1C1C;
```

Comment:
it's better to have values coming from the theme rather than being fixed values.
especially typography.

Not a necessary change since eventually we'll be moving to tailwind